controllers/cluster_controller_test.go

```diff
@@ -1101,10 +1101,9 @@ var _ = Describe(string(fdbtypes.ProcessClassClusterController), func() {
 					generations, err := reloadClusterGenerations(cluster)
 					Expect(err).NotTo(HaveOccurred())
 					Expect(generations).To(Equal(fdbtypes.ClusterGenerationStatus{
-						Reconciled:             originalVersion,
-						NeedsBounce:            originalVersion + 1,
-						NeedsMonitorConfUpdate: originalVersion + 1,
-						HasUnhealthyProcess:    originalVersion + 1,
+						Reconciled:          originalVersion,
+						NeedsBounce:         originalVersion + 1,
+						HasUnhealthyProcess: originalVersion + 1,
```

Comment:
Was the previous state incorrect?

Reply:
I think the previous state was reflecting reconciliation not making it as far as it ideally could in this configuration.

Reply:
Oh right 👍